package.json

```diff
@@ -1,6 +1,8 @@
 {
 	"dependencies": {
 		"@babel/parser": "^7.20.3",
-		"@babel/types": "^7.20.2"
+		"@babel/types": "^7.20.2",
+		"express-fileupload": "^1.5.0"
```

Comment:
@MallanagoudaB remove file upload package from here and add it in the src/package.json